src/Header/MenuButton/MenuButton.js

```diff
@@ -0,0 +1,9 @@
+const MenuButton = (props) => {
+    const toggleMenu = () => {
+        props.sidebarIsHidden === false ? props.setSidebarIsHidden(true) : props.setSidebarIsHidden(false)
```

Comment:
this is quite long, whereas with booleans you can do:
```js
props.setSidebarIsHidden(!props.sidebarIsHidden)
```

Reply:
so it sets it to the opposite of the current value

Reply:
that is so clean....